src/components/TodoItem.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TodoItem = ({ id, title, completed, onToggleCompleted, onDeleted }) => (
```

Comment:
```suggestion
const TodoItem = ({
  id, title, completed, 
  onToggleCompleted, onDeleted
}) => (
```
Разбивай длинные строки на более короткие